third_party/proton/README.md

```diff
@@ -232,12 +302,7 @@ with proton.scope("test"):
 
 The call path of `foo1` will be `test->test1->state0`.
 
-### Instrumentation (experimental)
-
-In addition to profiling, Proton also incorporates MLIR/LLVM based compiler instrumentation passes to get Triton level analysis
-and optimization information. This feature is under active development and the list of available passes is expected to grow.
-
-#### Available passes
+### LLVM Instrumentation (deprecated)
```

Comment:
Are we planning on removing this?

Reply:
Yeah, I think it should be merged into our current instrumentation framework, at least from proton side